README.md

```diff
@@ -18,13 +18,8 @@ installed. If you want to use tools-deps, specify either `clojure:tools-deps`,
 
 ### boot
 
-Prior to JDK 20 we provided [boot](https://boot-clj.github.io) variants as well. 
-Boot hasn't had a release since 2019, and it is breaking in more and more image
-variants. Boot variants are now deprecated and no new boot images will be
-provided for JDK 20+, it will no longer be installed in the `latest` image, and
-no alpine-based images will be provided for any JDK version (due to breakage).
-As long as the image builds don't break, we will continue providing boot images
-for non-alpine distros for JDK 17 and lower until those versions are EOL'd.
+New [boot](https://boot-clj.github.io) images are no longer provided. The
```

Comment:
Can you add e.g. "As of 5/2024" to this so folks know roughly when the change was made?